genome_browser/_tracks.py

```diff
@@ -16,11 +16,11 @@
 
 
 class Track(object):
-    def __init__(self, name=None, height_ratio=1):
+    def __init__(self, name="", height_ratio=1, step=500):
         self.annotate = True
         self.height_ratio = height_ratio
         self.name = name
-        self.step = 500
+        self.step = step
```

Comment:
The `500` default is awesome for genomics, but specifying what one needs wins for me.
It also helped making the demo script look nicer out of the box :+1: